sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRelation.scala

```diff
@@ -111,17 +113,20 @@ private[sql] class DefaultSource extends RelationProvider {
         numPartitions.toInt)
     }
     val parts = JDBCRelation.columnPartition(partitionInfo)
-    JDBCRelation(url, table, parts)(sqlContext)
+    val properties = new Properties() // Additional properties that we will pass to getConnection
```

Comment:
(i.e. http://spark.apache.org/docs/latest/sql-programming-guide.html#jdbc-to-other-databases)